include/deflation.h

```diff
@@ -62,7 +66,7 @@ namespace quda {
         //Check that RV is a composite field:
         if(RV->IsComposite() == false) errorQuda("\nRitz vectors must be contained in a composite field.\n");
      }
-
+    
```

Comment:
Why are these changes in this file?

Reply:
Looks like a whitespace. I can remove if needed.

Reply:
This looks like a line got duplicated. If this is the only difference I will revert to the GK file

Reply:
reverted in e0e2c71f1 - to be pushed.